requirements/torch_rocm62.txt

```diff
@@ -0,0 +1,4 @@
+--index-url https://download.pytorch.org/whl/rocm6.2
+torch==2.5.1+rocm6.2; sys_platform == 'linux'
+torchvision==0.20.1+rocm6.2; sys_platform == 'linux'
+torchaudio==2.5.1+rocm6.2; sys_platform == 'linux'

```

Comment:
We haven't yet updated the Pytorch version for the rest of the project. But this should be ok. I will update it for other platforms too